recipes/pytfa/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "pytfa" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/EPFL-LCSB/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  #url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b4d848bd04cf8384b05e82fecb25a7cc59a65031ac5dee2c732853a12b086014
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+
+  run:
+    - python >=3.8
+    - equilibrator-api
+    - equilibrator-cache
+    - sympy <=1.5
```

Comment:
This version of pytfa requires some `sympy >1.5`. Why did you choose `<=1.5`? The upstream has no version requirement.

https://github.com/EPFL-LCSB/pytfa/blob/master/requirements.txt

Reply:
Yes in the latest release of pytfa it is not listed as a requirement but critical functions break for sympy > 1.5 due to an import error from sympy. This should be resolved in a future release.